worker/deploy/README.md

```diff
@@ -1,47 +1,123 @@
-# Worker Node deployment
+# Worker Node Deployment
 
-Arrebol tasks are processed by the Workers. In a typical deployment, multiple Workers are deployed together in a single virtual machine, a `Worker Node`. As each task run in an isolated docker container, the Docker engine and other dependencies must have be installed and configured in the Worker Node. Below, we detail the configuration procedure.
+## **Overview**
 
-## Requeriments
+### _Worker Node_
 
-Before the configuration and installation of Worker Node dependencies, each Worker Node virtual machine should be configured to be reached via SSH (using a rsa key pair). Also, the [Ansible](https://www.ansible.com/) automation tool should be installed in the deploy `coordination host`. This host is a machine that commands that deploy. To this end, the `coordination host` should be able to access the Worker Node through SSH.
+**Worker Node** is the host where workers live. You need to deploy at least one worker node for Arrebol.
 
-To install Ansible run these commands:
+### _Coordination Host_
+
+The **coordination host** is basically a machine from which scripts are executed to deploy worker nodes. The coordination host must have access to worker nodes via **SSH (using a rsa key pair)** to perform its function.
+
+![Worker Deployment](../../imgs/wd.png)
+
+## **Minimum Requirements**
+
+### _Worker Node_
+  The requirements of the Worker Node depend a lot on the workload of the Jobs you intend to submit to Arrebol. Below is a minimum standard recommendation:
+
+  - vCPU: 2
+  - RAM: 2GB
+  - FREE DISK SPACE: 10GB
+  - OS: Ubuntu 16 or higher
+
+### _Coordination Host_
+  - vCPU: 1
+  - RAM: 1GB
+  - FREE DISK SPACE: 10GB
+  - OS: Ubuntu 16 or higher
+
+## **Setup**
+
+### _1.Create RSA Key Pair_
+
+`Note: If you already have a key pair configured between the coordination host and the worker nodes go to Install dependencies.`
+
+Log into the coordination host, and then use command-line SSH to generate a key pair using the RSA algorithm.
+To generate RSA keys, on the command line, enter:
+
+```bash
+mkdir -p ~/.ssh
+cd ~/.ssh
+ssh-keygen -t rsa -N "" -f coordination_host_key
+chmod 600 coordination_host_key
+```
+The coordination host private key will be generated using the filename 
+**coordination_host_key** and the public key using the filename **coordination_host_key.pub**. It was stored in a **.ssh** directory off home directory.
+
+Now you need to authorize the coordination host to access the worker node via SSH.
+
+First you need have the **coordination_host_key.pub** in the worker node. After, log into the worker node machine and write the **coordination_host_key.pub** into **authorized keys** using the commands below:
 ```bash
+mkdir -p ~/.ssh
+touch ~/.ssh/authorized_keys
+cat ~/coordination_host_key.pub >> ~/.ssh/authorized_keys
```

Comment:
only if the public key is in the home directory, right? One maybe have the key stored on ctrl+c at the moment, so it only have to open the authorized_keys file and paste it there... I think you could refactor this subsection by explaining that what is important is to save the public key in the authorized key file, and, this is an example about how to do that if the public key is in the home directory

Reply:
Done.